posthog-core/src/types.ts

```diff
@@ -258,3 +256,231 @@ export type EvaluationReason = {
   condition_index: number | undefined
   description: string | undefined
 }
+
+// survey types
```

Comment:
moved all types from survey-types to types to avoid importing/missing module issue